src/main/java/net/minestom/server/event/inventory/InventoryItemChangeEvent.java

```diff
@@ -54,11 +53,14 @@ public int getSlot() {
      * @return a new item on a changed slot.
      */
     public @NotNull ItemStack getNewItem() {
-        return newItem;
+        return itemStacks[1];
     }
 
     @Override
     public @Nullable Inventory getInventory() {
         return inventory;
     }
+
```

Comment:
Consider adding some javadoc here (size of the array, indexes of the contents)

Reply:
Sure I’ll do that tomorrow. Thanks for the feedback 😄